src/Autocomplete/tests/Unit/AutocompleteResultsExecutorTest.php

```diff
@@ -15,8 +15,9 @@
 use Doctrine\ORM\EntityRepository;
 use Doctrine\ORM\QueryBuilder;
 use PHPUnit\Framework\TestCase;
-use Symfony\Bundle\SecurityBundle\Security\Security;
+//use Symfony\Bundle\SecurityBundle\Security;
 use Symfony\Component\Security\Core\Exception\AccessDeniedException;
+use Symfony\Component\Security\Core\Security;
```

Comment:
What's going on here? Playing with making things work on multiple Symfony versions?

Reply:
yes, the namespace has changed on newer versions..